.github/workflows/test-and-deploy.yml

```diff
@@ -187,9 +213,26 @@ jobs:
     - should-deploy
     - unit-test
     - integration-test
+    - build-docker-image
+
     if: needs.should-deploy.outputs.should-run == 'true'
 
     steps:
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GHCR_PERSONAL_ACCESS_TOKEN }}
+    - name: Cache Docker layers
+      uses: satackey/action-docker-layer-caching@v0.0.11
+      continue-on-error: true
+    - name: Pull existing image
+      run: docker pull ghcr.io/ualbertaaltlab/itwewina.altlab.app:${{ github.sha }}
+    - name: Tag previously successfully created image as latest
+      run: docker tag ghcr.io/ualbertaaltlab/itwewina.altlab.app:${{ github.sha }} ghcr.io/ualbertaaltlab/itwewina.altlab.app:latest
+    - name: Update the latest tag
+      run : docker push ghcr.io/ualbertaaltlab/itwewina.altlab.app:latest
```

Comment:
Is there a better way of retagging the image _without_ pulling the image and running `docker tag` and `docker push`?